### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by lolobosse**
Hello there,

Thanks for your work 👍 

I made some small changes:
 - Formatted code
 - Made every function to arrow function to remove confusing `bind`
 - Downloaded the asset, not to be dependent on third-party server
 - Added the `event.preventDefault()` on the button click in order to prevent form submission if this list is in a `<form>` (it is my case)
 - Linked the demo to the actual code in order to have a development capable demo.

You can, of course, cherry pick the changes you want, I will use my clone first 😄

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,9 @@
 {
-    "presets": ["@babel/preset-env", "@babel/preset-react"]
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties"
```
Comment (arpitbatra123) [inline .babelrc]: What does this babel plugin do?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
 node_modules
 .cache
 dist
+.idea
+.idea/
```
Comment (arpitbatra123) [inline .gitignore]: Does `.idea` without the backslash not serve the purpose of ignoring the folder as well as it's contents?

File: assets/close.svg [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (arpitbatra123) [inline assets/close.svg]: I optimized this SVG using https://jakearchibald.github.io/svgomg/ and the size reduced to half (612 bytes → 324 bytes). We can replace the SVG with the optimized version:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 348.333 348.334">
  <defs/>
  <path d="M336.559 68.611L231.016 174.165l105.543 105.549c15.699 15.705 15.699 41.145 0 56.85-7.844 7.844-18.128 11.769-28.407 11.769-10.296 0-20.581-3.919-28.419-11.769L174.167 231.003 68.609 336.563c-7.843 7.844-18.128 11.769-28.416 11.769-10.285 0-20.563-3.919-28.413-11.769-15.699-15.698-15.699-41.139 0-56.85l105.54-105.549L11.774 68.611c-15.699-15.699-15.699-41.145 0-56.844 15.696-15.687 41.127-15.687 56.829 0l105.563 105.554L279.721 11.767c15.705-15.687 41.139-15.687 56.832 0 15.705 15.699 15.705 41.145.006 56.844z"/>
</svg>
```

File: demo/index.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-import EditableList from 'react-list-editable';
+import EditableList from '../src';
```
Comment (arpitbatra123) [inline demo/index.js]: 💯

File: .babelrc (hunk 1 above)
Comment (lolobosse) [reply .babelrc]: It allows the arrow functions, if you remove it, you may not be able to compile

File: .gitignore (hunk 2 above)
Comment (lolobosse) [reply .gitignore]: Might be, you can correct that :)

File: assets/close.svg (hunk 3 above)
Comment (lolobosse) [reply assets/close.svg]: Cool! Yes let's do it

**Comment by arpitbatra123** [comment]
Merged. Thanks for the changes!

**Pull request merged**